backend/src/Application.ts

```diff
@@ -47,21 +47,14 @@ export default class Application {
   }
 
   private startHeartbeat = () => {
-    let count = 0
-    setInterval(() => {
-      this.check(count++)
-      if (count > 999) count = 0
-    }, 1000 * 60) // 1bpm
+    setInterval(this.check, 1000 * 60) // 1bpm
```

Comment:
I feel these kinds of "magic" variables should be globals set in a common place. Thoughts?

Reply:
The interval check time... yea, I'll add to constants...